### ISSUE TITLE
Ignore loader chaining

### ISSUE THREAD

**Issue by ryuran** [nicolasRdr/twing-loader#2]
Hello

https://github.com/nicolasRdr/twing-loader/blob/master/src/loader.js#L15
This line use `this.ressource` it’s the path of original file and not the result of previous loaders.

We should une `sourceData` here
```
module.exports = async function loader(sourceData) {
```
But I don't know how use it with Twing API @ericmorand

**Comment by DerekRoth** [comment]
yes this is indeed a quick and imperfect Twing loader implementation. Using the result of previous loaders is apparently more complicated than it seems because of the fact that Twig requires file names for many features. If you can think of a fix, feel free to submit a PR :)

**Comment by ericmorand** [comment]
@ryuran could you please provide an example of webpack config that fails? I'd like to add it the the integration test suite of my loader - and hopefully next version of this one.

**Comment by ericmorand** [comment]
@ryuran Could you give twing-loader@2 a try?

**Comment by ryuran** [comment]
As soon as i'm back from vacation

**Comment by ryuran** [comment]
@ericmorand I do not try yet but by code reading I don't see reference to `this.resource` so what if I use a non-twig file as resource and use a loader to transform it to twig before twing-loader.

I can see, your loader always use the file from `this.resourcePath` and never use the content from previous loader transformation.

Maybe we should test with a simple fake loader returning a twig simple content and chain it before Twing-loader to see if it works:

```js
function(source) {
  return 'Twig from fake loader {{ variable }}';
}
```

I currently don't see loader chaining in you Twing-loader tests

**Comment by ryuran** [comment]
ho damn. my vacation wasn't so long…
I don't work anymore with twig/twing.
